automation_tools/__init__.py

```diff
@@ -1519,6 +1521,20 @@ def katello_installer(debug=True, sam=False, verbose=True, **kwargs):
     ))
 
 
+def katello_service(action, exclude=None):
```

Comment:
It is used for excluding the services you don't want to perform the action against. It's the only option provided by `katello-service --help`